resip/dum/Profile.hxx

```diff
@@ -236,6 +236,10 @@ class Profile
       virtual bool getExtraHeadersInReferNotifySipFragEnabled() const;
       virtual void unsetExtraHeadersInReferNotifySipFragEnabled();
 
+      virtual void setHandling491AsGeneralFailure(bool enabled);
```

Comment:
To follow suit on the naming scheme already in place, and to reflect that this is for InviteSessions only, please rename this and the member variable to be:
setHandleInviteSession491AsGeneralFailureEnabled